core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -694,8 +689,9 @@ public void statusChanged(PlaybackServiceMediaPlayer.PSMPInfo newInfo) {
                     }
                     break;
 
-                case ERROR:
+                case ERROR: case STOPPED:
```

Comment:
this is both formatted badly and a bad practice
I usually either add a `// deliberate fall-through` comment (only slightly better) or try to avoid this alltogether. For example, you could extract write.. and stop into a method with a proper name and use it in both cases separately

Reply:
I agree with @mfietz , especially considering that `STOPPED` and `ERROR` do not sound the same case intuitively.

I am also slightly worried if the change would break some other cases. Looking at the file history, the change effectively undid what a commit did 5 years ago.

Having two `case`s in the single line is also inconsistent with the style used in the same class, e.g., [line 504-505](
https://github.com/ByteHamster/AntennaPod/blob/c2ace741563f33d507ae7fd425ad659298cbd655/core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java#L505).